README.md

```diff
@@ -18,6 +18,8 @@ FDC3 is a standard.  This repository houses the documentation and specifications
 4. Push to the branch (`git push origin feature/fooBar`)
 5. Create a new Pull Request
 
+The team will review the PR and decide about merging. Note that you will require a contributor agreement (individual or corporate) before your code can be merge, so please review [FINOS Contribution requirements](.github/CONTRIBUTING.md).
```

Comment:
I was wondering yesterday why this CONTRIBUTING.md document is hidden away in the .github folder. In my opinion it should be in the root.

Reply:
Agreed! But I thought that was your choice so at least I linked it for now :) 

Feel free to reject this and do the "right thing" if you prefer.